erpnext/controllers/accounts_controller.py

```diff
@@ -1412,6 +1412,38 @@ def validate_and_delete_children(parent, data):
 	for d in deleted_children:
 		update_bin_on_delete(d, parent.doctype)
 
+
+@frappe.whitelist()
+def make_quality_inspections(doctype, docname, items):
```

Comment:
Accounts controller does not seem like the best location. Quality Inspection is carried out on stock/inventory and has no accounting impact. We can place this in `quality_inspection.py` itself perhaps.

Reply:
Ah, you're right. I can move this to the stock controller or quality inspection. Either way works.